### PR TITLE
Legolistfix

### PR DESCRIPTION
**Pull request by dougsillars**
## Description
added more features to the GitHb action listing the runbooks and actions.

Every action on readme pages now has a link to the action.

We also generate 2 new pages - a list of all the actions and all the runbooks. 

A github action pushes these two pages into the docs so they can also be published automatically in the documentation

### CODE REVIEW

File: generate_readme.ipynb [hunk 1]
```diff
@@ -36,7 +36,7 @@
   },
```
Comment (abhishek-unskript) [inline generate_readme.ipynb:36]: <pre class="ql-syntax" spellcheck="false">Line #45.    #print(actions)
</pre><p>Remove these lines (test commenting from reviewnb)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/unskript/Awesome-CloudOps-Automation/pull/119/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

File: AWS/README.md [hunk 2]
```diff
@@ -38,50 +38,337 @@
  # AWS Actions 
 
 * Aws apply default encryption for s3 bucket.py
+* Aws attach ebs to instances.py
```
Comment (jayasimha-raghavan-unskript) [inline AWS/README.md:41]: Are these Links? I am not seeing that in here. Appears like a file name (First letter capitalized).py

File: AWS/README.md [hunk 3]
```diff
@@ -38,50 +38,337 @@
...
 * Aws restart instances.py
 * Aws run instances.py
 * Aws stop instances.py
+* Aws target group list instances.py
+* Aws target group list unhealthy instances.py
+* Aws target group register unregister instances.py
 * Aws terminate ec2 instances.py
 * Aws update access keys.py
+* Aws upload file to s3.py
+
```
Comment (jayasimha-raghavan-unskript) [inline AWS/README.md:149]: Got it. Guess the above section can be removed from the README.md?

File: ElasticSearch/README.md [hunk 4]
```diff
@@ -3,3 +3,48 @@
  # ElasticSearch xRunBooks
 
 * [Elasticsearch Rolling Restart](https://github.com/unskript/Awesome-CloudOps-Automation/tree/master)
+
+
+ # ElasticSearch Actions 
+
+* Elasticsearch check health status.py
```
Comment (jayasimha-raghavan-unskript) [inline ElasticSearch/README.md:10]: We can delete these section, just like in AWS Section.

File: ElasticSearch/README.md [hunk 5]
```diff
@@ -3,3 +3,48 @@
...
+| ElasticSearch | [Elasticsearch enable shard allocation.py](https://github.com/unskript/Awesome-CloudOps-Automation/tree/master/ElasticSearch/legos/elasticsearch_enable_shard_allocation) 
+| ElasticSearch | [Elasticsearch get handle.py](https://github.com/unskript/Awesome-CloudOps-Automation/tree/master/ElasticSearch/legos/elasticsearch_get_handle) 
+| ElasticSearch | [Elasticsearch list allocations.py](https://github.com/unskript/Awesome-CloudOps-Automation/tree/master/ElasticSearch/legos/elasticsearch_list_allocations) 
+| ElasticSearch | [Elasticsearch list nodes.py](https://github.com/unskript/Awesome-CloudOps-Automation/tree/master/ElasticSearch/legos/elasticsearch_list_nodes) 
+| ElasticSearch | [Elasticsearch search query.py](https://github.com/unskript/Awesome-CloudOps-Automation/tree/master/ElasticSearch/legos/elasticsearch_search_query) 
+| ElasticSearch | [Elasticsearch search query new.py](https://github.com/unskript/Awesome-CloudOps-Automation/tree/master/ElasticSearch/legos/elasticsearch_search_query_new) 
+
```
Comment (jayasimha-raghavan-unskript) [inline ElasticSearch/README.md:36]: Why we have two entries?

File: Jenkins/README.md [hunk 6]
```diff
@@ -3,3 +3,24 @@
  # Jenkins xRunBooks
 
 * [Fetch Jenkins Build Logs](https://github.com/unskript/Awesome-CloudOps-Automation/tree/master)
+
+
```
Comment (jayasimha-raghavan-unskript) [inline Jenkins/README.md:7]: Remove this section

File: Jenkins/README.md [hunk 7]
```diff
@@ -3,3 +3,24 @@
...
+ # Jenkins Actions 
+
+| Jenkins | [Jenkins get handle.py](https://github.com/unskript/Awesome-CloudOps-Automation/tree/master/Jenkins/legos/jenkins_get_handle) 
+| Jenkins | [Jenkins get logs.py](https://github.com/unskript/Awesome-CloudOps-Automation/tree/master/Jenkins/legos/jenkins_get_logs) 
+| Jenkins | [Jenkins get plugins.py](https://github.com/unskript/Awesome-CloudOps-Automation/tree/master/Jenkins/legos/jenkins_get_plugins) 
+
+
+ # Jenkins Actions
```
Comment (jayasimha-raghavan-unskript) [inline Jenkins/README.md:22]: Looks like duplicate entry. May be common problem in the ipynb file?

**Review by jayasimha-raghavan-unskript — CHANGES_REQUESTED** [review]
Please take care of the comments.

File: AWS/README.md (hunk 2 above)
Comment (dougsillars) [reply AWS/README.md]: this was a bug in the Script - its fixed now

File: AWS/README.md (hunk 3 above)
Comment (dougsillars) [reply AWS/README.md]: fixed... one time issue as we change the code,

File: ElasticSearch/README.md (hunk 5 above)
Comment (dougsillars) [reply ElasticSearch/README.md]: one is called _new.  Should we delete the "old" one?

**Pull request merged**